runners/spark/src/main/java/org/apache/beam/runners/spark/io/SparkUnboundedSource.java

```diff
@@ -148,18 +155,25 @@ private static void checkpointStream(JavaDStream<?> dStream,
    * <p>Updates {@link MetricsAccumulator} with metrics reported in the read.</p>
    */
   private static class ReadReportDStream extends DStream<BoxedUnit> {
+
+    private static final String READ_DURATION_MILLIS = "readDurationMillis";
+    private static final String NAMESPACE = "io";
```

Comment:
Since this is a Spark-runner specific metric, perhaps we should put it in a namespace that indicates this, something like `spark_metrics`